.gitignore

```diff
@@ -22,3 +22,10 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+# Archivos de entorno
+.env
+.env.local
+.env.*.local
+.env.production
+.env*
```

Comment:
`.env*` should be enough, you don't have to explicitly list all of them one by one.